packages/expect/src/__tests__/matchers.test.js

```diff
@@ -271,6 +271,16 @@ describe('.toStrictEqual()', () => {
     expect(c.constructor.name).toEqual(d.constructor.name);
     expect({test: c}).not.toStrictEqual({test: d});
   });
+
+  it('does not pass when sparseness of arrays do not match', () => {
+    expect([, 1]).not.toStrictEqual([undefined, 1]); // eslint-disable-line no-sparse-arrays
```

Comment:
Can you stick a `/* eslint-disable no-sparse-array */` above line 275, and a `/* eslint-enable */`  below line 283 instead if repeating it?

Reply:
Yes, good call. Done in https://github.com/facebook/jest/pull/7591/commits/fe65db71ee364b4516eb617a851601aace5393aa